src/Raven.Server/Documents/Handlers/AI/Agents/ConversationHandler.cs

```diff
@@ -250,6 +262,23 @@ private async Task SummarizeAsync(JsonOperationContext context, ChatCompletionCl
             : summarization.SummarizationTaskBeginningPrompt;
         beginningPrompt += $" The original system prompt was: {systemPrompt}, the rest of follows";
 
+        var endPrompt = string.IsNullOrEmpty(summarization.SummarizationTaskEndPrompt)
+            ? database.Configuration.Ai.SummarizationTaskEndPrompt
+            : summarization.SummarizationTaskEndPrompt;
+
+        if (EndsWithTool(oldChat))
+        {
+            // Add a small nudge when the transcript ends with tool output, in order to avoid running the same tool again
+
+            beginningPrompt +=
+                " The transcript ends with tool output. Your summary must explicitly include the key results from those tool calls. " +
+                "These results must appear in the assistant’s summary message so that the conversation can continue without re-running the same tools.";
+
+            endPrompt +=
```

Comment:
why not have those as default prompts? 
If the user want to change those defaults he still be forced to have those..